crypto/.jvm/src/main/scala/org/bitcoins/crypto/BouncyCastleUtil.scala

```diff
@@ -26,16 +26,16 @@ object BouncyCastleUtil {
     decodePubKey(point, publicKey.isCompressed)
   }
 
-  def decodePoint(bytes: ByteVector): ECPoint = {
+  def decodePoint(bytes: ByteVector): org.bouncycastle.math.ec.ECPoint = {
```

Comment:
why is this better than an import?

Reply:
if the problem is collisions with the `crypto.ECPoint` why not just import this as `BouncyCastlePoint` or something like this?

Reply:
Because it works! TM